api/migrations/0002_data_migration.py

```diff
@@ -97,6 +99,66 @@ def _write_languages_and_levels(self):
         )
         LanguageAndLevel.objects.bulk_create(language_and_level_objects)
 
+    def _write_enrollment_tests(self):
+        """Writes `EnrollmentTest', `Question`, and `EnrollmentTestQuestionOption` to database."""
+        EnrollmentTest = self.apps.get_model(APP_NAME, "EnrollmentTest")
+        EnrollmentTestQuestion = self.apps.get_model(APP_NAME, "EnrollmentTestQuestion")
+        EnrollmentTestQuestionOption = self.apps.get_model(
+            APP_NAME, "EnrollmentTestQuestionOption"
+        )
+
+        AgeRange = self.apps.get_model(APP_NAME, "AgeRange")
+        Language = self.apps.get_model(APP_NAME, "Language")
+
+        path = (
+            settings.BASE_DIR
+            / APP_NAME
+            / "migrations"
+            / "data_migration_sources"
+            / "enrollment_tests.yaml"
+        )
+        with path.open(encoding="utf8") as fh:
+            data = yaml.safe_load(fh)
+
+        for block in data:
+            enrollment_test = EnrollmentTest.objects.create(
+                language=Language.objects.get(id=block["language"]),
```

Comment:
I think we can skip object getting by setting language_id directly

Reply:
Right, since id=locale for Language. Fixed